react/features/virtual-background/functions.js

```diff
@@ -52,28 +53,35 @@ export const toDataURL = async (url: string) => {
  * @param {Object} base64image - Base64 image extraction.
  * @param {number} width - Value for resizing the image width.
  * @param {number} height - Value for resizing the image height.
- * @returns {Object} Returns the canvas output.
+ * @returns {Promise<string>}
  *
  */
-export async function resizeImage(base64image: any, width: number = 1920, height: number = 1080) {
-    const img = document.createElement('img');
+export async function resizeImage(base64image: any, width: number = 1920, height: number = 1080): Promise<string> {
```

Comment:
I think you can skip the "async" here since you return a Promise. Check if it's harmless please.